test/e2e2/conftest.py

```diff
@@ -27,30 +32,87 @@ def random_email() -> str:
     def random_password() -> str:
         return "".join(random.choice(string.ascii_letters) for _ in range(16))
 
+    @staticmethod
+    def random_domainname():
+        return (
+            "".join(random.choice(string.ascii_lowercase) for _ in range(16))
+            + ".test"
+        )
+
+    @staticmethod
+    def random_local_public_suffix_domainname():
+        return (
+            "".join(random.choice(string.ascii_lowercase) for _ in range(16))
+            + ".dedyn."
+            + os.environ['DESECSTACK_DOMAIN']
+        )
```

Comment:
Why are these methods of the client (whose instances are usually labeled `api_user`, so you get `api_user.random_domainname()`)?

Reply:
converted to fixtures